.github/workflows/test.yml

```
@@ -15,7 +15,7 @@ concurrency:
 env:
   POLYCLI_VERSION: v0.1.64 # https://github.com/0xPolygon/polygon-cli/releases/tag/v0.1.64 - 2024-11-25
   # Use fork instead of cdk repository until they migrate to the new version of kurtosis-cdk.
-  CDK_VERSION: d9845e94bf3c93fb297f81942e1673bc01fd2701 # https://github.com/leovct/cdk/commit/d9845e94bf3c93fb297f81942e1673bc01fd2701
+  CDK_VERSION: develop # https://github.com/0xPolygon/cdk/tree/develop
```

We'd like to make sure our CI does not break because of a new commit. Can you use this specific [commit](https://github.com/0xPolygon/cdk/commit/e254ec910be055fbc73b33d21bb99d7c492a0ac0) (the latest commit on `develop`)?

```suggestion
  CDK_VERSION: e254ec910be055fbc73b33d21bb99d7c492a0ac0 # https://github.com/0xPolygon/cdk/tree/develop
```